code/modules/jobs/job_types/bridge.dm

```diff
@@ -16,10 +16,10 @@ Bridge Officer
 
 	outfit = /datum/outfit/job/bofficer
 
-	access = list(access_heads, access_helm, access_maint_tunnels)
-	minimal_access = list(access_heads, access_helm, access_maint_tunnels)
+	access = list(GLOB.access_heads, GLOB.access_helm, GLOB.access_maint_tunnels)
+	minimal_access = list(GLOB.access_heads, GLOB.access_helm, GLOB.access_maint_tunnels)
 
-GLOBAL_LIST_INIT(posts = list("weapons", "helms"))
+var/list/posts = list("weapons", "helms")
```

Comment:
it's a global list either way, making it use GLOB instead allows admins to look at it in game